config/nrfconnect/overlay-sample-defaults.conf

```diff
@@ -0,0 +1,83 @@
+#
```

Comment:
We should only use CHIP copyright header when we contribute to CHIP, I think

Reply:
Ok, but solution was pretty much copy+pasted from NCS OpenThread samples, so maybe it would be worth to mention it somewhere?

Reply:
OK, but CHIP code owners will demand that you add CHIP copyright header as well, so you may use the CHIP copyright template and add a line mentioning Nordic Semiconductor. See AppEvent.h files copy-pasted from Nest Lab projects.

Reply:
Ok, done